openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/ManageButton/ManageButton.tsx

```diff
@@ -93,6 +97,45 @@ const ManageButton: FC<Props> = ({
           ),
           key: 'delete-button',
         },
+        ...(deleted
+          ? [
+              {
+                label: (
+                  <Tooltip title={canDelete ? '' : NO_PERMISSION_FOR_ACTION}>
+                    <Space
+                      className={classNames('tw-cursor-pointer manage-button', {
+                        'tw-cursor-not-allowed tw-opacity-50': !canDelete,
+                      })}
+                      size={8}
+                      onClick={(e) => {
+                        if (canDelete) {
+                          e.stopPropagation();
+                          setShowActions(false);
+                          onRestoreEntity && onRestoreEntity();
+                        }
+                      }}>
+                      <SVGIcons alt="Restore" icon={Icons.RESTORE} />
+                      <div
+                        className="tw-text-left"
+                        data-testid="restore-button">
+                        <p
+                          className="tw-font-medium"
+                          data-testid="delete-button-title">
+                          Restore
+                        </p>
+                        <p className="tw-text-grey-muted tw-text-xs">
+                          Restoring this {entityType} will restore its metadata
```

Comment:
Can we add localisation for new strings?

Reply:
@chirag-madlani Localisation added